asterius/test/primitive/primitive.hs

```diff
@@ -0,0 +1,166 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
I assume the unit test is just a WIP? The `primitive` stuff is not called at all and the pasted `fib` logic is not cleaned up.

Reply:
Yeah, sorry about that, I had already committed locally so it came along :woman_facepalming: